LondonBritishLibrary/orient/BLorient8815.xml

```diff
@@ -0,0 +1,274 @@
+                                    <msItem xml:id="ms_i1.2.1.6">
+                                        <locus from="22r" to="23r"/>
+                                        <title ref="LIT1828Mahale#Manasseh"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.7">
+                                        <locus target="#23r"/>
+                                        <title ref="LIT1828Mahale#Jonah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.8">
+                                        <locus from="23r" to="24v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths1"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.9">
+                                        <locus target="#24v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths2"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.10">
+                                        <locus from="24v" to="25v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths3"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.11">
+                                        <locus from="25v" to="27r"/>
+                                        <title ref="LIT1828Mahale#Habakkuk"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.12">
+                                        <locus from="27r" to="27v"/>
+                                        <title ref="LIT1828Mahale#Isaiah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.13">
+                                        <locus from="27v" to="28r"/>
+                                        <title ref="LIT1828Mahale#Mary"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.14">
+                                        <locus from="28v" to="28v"/>
+                                        <title ref="LIT1828Mahale#Zachariah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.15">
+                                        <locus from="28v" to="29r"/>
+                                        <title ref="LIT1828Mahale#Simeon"/>
+                                    </msItem>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2">
+                                    <locus from="94r" to="97v"/>
+                                    <title ref="LIT2362Songof" type="incomplete"/>
+                                    <note>The text is divided into five sections. Desinit ex abrupto (VIII, 2).</note>
```

Comment:
I think it would be kind towards many users to reformulate "Desinit ex abrupto" in english (or give an equivalent in parantheses, if you prefer to stay close to the wording of the catalogue)